ZUT-PokerPlanningClient/src/app/project-planning/game/game.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { environment } from 'src/environments/environment';
+import { Observable } from 'rxjs';
+import { HttpClient } from '@angular/common/http';
+import { UserItem } from './user-item';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class GameService {
+  private httpClient: HttpClient;
+  private baseURL = environment.baseUrl;
+
+  constructor(httpClient: HttpClient) {
+    this.httpClient = httpClient;
+  }
+
+  getGameUsers(): Observable<UserItem[]> {
```

Comment:
Logically I'd suggest to create UsersService with this logic and GameService would use UsersService, but thats fine at the moment!